### ISSUE TITLE
[A11y] - Plugin System - Additional Accessibility Features is not correctly detected in the page landmarks by Skip To plugin

### ISSUE THREAD

**Issue by carcam** [joomla/joomla-cms#34401]
It seems the new Plugin System - Additional Accessibility Features is not correctly detected in the page landmarks by the "Skip to" plugin

### Steps to reproduce the issue
1. Go to plugin manager and enable Skip To plugin for frontend
2. Enable plugin System - Additional Accessibility Features for Frontend
3. Go to the frontend of your page
4. Use your keyboard (press Tab Key) to reach the *Keyboard navigation* menu
5. Open it clicking Enter

### Expected result

In the page outline feature we should see a "Toggle" accessibility options and after choosing it, the focus will be on the Accessibility Options Button.

### Actual result

In the page outline we see an entry called "Close Accesibility Options..." and when reaching it with your keyboard it does not go to the accessibility icon.

![Captura de pantalla 2021-06-04 a las 17 05 19](https://user-images.githubusercontent.com/1695138/120823195-79ba2100-c557-11eb-8dbd-110cdb9290af.png)

### System information (as much as possible)

- Latest Joomla from GH after npm ci and php composer update

@joomla/joomla-accessibility-team-jat

**Comment by brianteeman** [comment]
The skipto plugin is not configured to show everything - it would be very very long if it displayed everything that is available.

Currently it is configured 

```
						// Selectors for landmark and headings sections
						'headings'  => 'h1, h2, h3',
						'landmarks' => 'main, nav, search, aside, header, footer, form',
```

For more information on this see https://github.com/paypal/skipto#css-selectors-for-identifying-landmarks-and-headings

**Comment by carcam** [comment]
Maybe we should include specifically the A11y options button or remove the markup that is adding that useless link... What do you suggest?

**Comment by brianteeman** [comment]
sorry just saw "the useless link" didnt see that before.

**Comment by drmenzelit** [comment]
Confirmed. The link in the skip-to plugin doesn't work, either to reach the accessibility button nor to close it if already open.

**Comment by chmst** [comment]
Confirmed
